infra/eligibility_screener/template/ecs.tf

```diff
@@ -11,6 +24,7 @@ resource "aws_security_group" "allow-screener-traffic" {
     protocol    = "tcp"
     cidr_blocks = ["0.0.0.0/8"]
   }
```

Comment:
Technically this ingress into port 80 is no longer needed since we're placing the container behind a load balancer 😄